core-bundle/src/Resources/contao/library/Contao/Email.php

```diff
@@ -321,9 +325,9 @@ public function replyTo()
 	 * @param string $strFile The file path
 	 * @param string $strMime The MIME type (defaults to "application/octet-stream")
 	 */
-	public function attachFile($strFile, $strMime='application/octet-stream')
+	public function attachFile($strFile, $strMime=null)
```

Comment:
#### Note

I changed the default value of `$strMime` to `null`, since `symfony/mime` will then automatically guess the correct MIME type. However, this might be considered a BC break and may be we should leave the default at `'application/octet-stream'`.

Reply:
I'd leave the default, let's not break BC for no reason. This class shouldn't really be used anymore anyway. Imho we should deprecate the whole class and ask users to go with the mailer service/component directly.

Reply:
Yeah, I was wondering why it was not deprecated yet anyway.